src/components/Display.js

```diff
@@ -1,18 +1,37 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-class Display extends React.Component {
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Kindly remove the disable linters, as linter rules help guide us to write industry standard code.